File: docs/examples/quickstart/ruby/expense.rb
class Expense
  attr_reader :amount, :description, :submitted_by

  def initialize(amount:, description:, submitted_by:)
    @amount = amount
    @description = description
    @submitted_by = submitted_by
  end
end

EXPENSES = {
  1 => Expense.new(amount: 500, description: "coffee", submitted_by:   "alice@example.com"),
  2 => Expense.new(amount: 5000, description: "software", submitted_by: "alice@example.com"),
  3 => Expense.new(amount: 50_000, description:"flight", submitted_by: "bhavik@example.com")
}

File: docs/examples/application/expenses-spring-boot/src/main/java/com/example/springboot/Expense.java
package com.example.springboot;

public class Expense {
    public int amount;
    public String description;
    public String submittedBy;

    public static Expense[] EXPENSES = { new Expense(500, "coffee", "alice@example.com"),
            new Expense(5000, "software", "alice@example.com"), new Expense(50000, "flight", "bhavik@example.com"), };

    public Expense(int amount, String description, String submittedBy) {
        this.amount = amount;
        this.description = description;
        this.submittedBy = submittedBy;
    }

    public static Expense lookup(int id) {
        if (id < EXPENSES.length) {
            return EXPENSES[id];
        } else {
            return null;
        }
    }

    public String toString() {
        return String.format("Expense(amount=%d, description=%s, submittedBy=%s)", this.amount, this.description,
                this.submittedBy);
    }
}